src/main/resources/templates/error/error.peb

```diff
@@ -0,0 +1,27 @@
+{# @pebvariable name="global" type="org.koil.view.GlobalViewModel" #}
+
+    {% extends 'layouts/dashboard-layout' %}
+    {% block title %} {{ pageTitle }} {% endblock %}
+    {% block meta %}
+    <meta name="turbolinks-visit-control" content="reload">
+    {% endblock %}
+    {% block scripts %}
+    <script>
+        function goBack() {
+            window.history.back()
+        }
+    </script>
```

Comment:
Can we extract this into a stimulus controller please? Inline scripts aren't really allowed anymore under modern content policies (to prevent XSS).

Reply:
Done.